pootle/apps/pootle_project/models.py

```diff
@@ -595,11 +600,21 @@ def invalidate_accessible_projects_cache(sender, instance, **kwargs):
 
     # FIXME: use Redis directly to clear these caches effectively
 
-    cache.delete_many([
-        make_method_key('Project', 'cached_dict', {'is_admin': False}),
-        make_method_key('Project', 'cached_dict', {'is_admin': True}),
-    ])
-
     User = get_user_model()
-    users_list = User.objects.values_list('username', flat=True)
+    users_list = User.objects.filter(is_superuser=False) \
+                             .values_list('username', flat=True)
+    cache.delete_many(
```

Comment:
Related to the `FIXME` note above, which assumes Redis SCANs will be quicker than querying the DB, using `cache.delete_pattern()` avoids having to retrieve all usernames when invalidating the cache ([see example](https://github.com/translate/pootle/blob/6b265e8b0564c18c1af429fac607ca297eafdc40/pootle/apps/pootle_language/models.py#L216-L217)). Note that many of the usernames might not even be present in the cache.